tardis/base.py

```diff
@@ -7,6 +7,7 @@ def run_tardis(
     packet_source=None,
     simulation_callbacks=[],
     virtual_packet_logging=False,
+    verbose_suppression=True,  # CHANGE1!! adding a paramenter to suppress the verbosity
```

Comment:
comments in the code are fine, and encouraged in places where the code may be particularly dense or confusing. However, comments referring to changes in the code will quickly become obsolete after the code is updated. Also, we can easily see the proposed changes in your PRs using GitHub's diff feature, that will highlight the modifications made. So... I would say that a comment detailing what the flag does is okay, but not also a comment numbering the change

Reply:
yes, it actually makes sense what you are trying to say, will update accordingly. thanks for rectifying